app/descriptors/org/questions.json

```diff
@@ -26,7 +26,11 @@
       "phase": "אושרו",
       "period": "כל השנים"
     },
-    "headers":["נושא","משרד","סוג תמיכה"]
+    "headers": ["נושא", "משרד", "סוג תמיכה"],
+    "formatters": [
+      { "סה״כ שולם": "budgetNumberFormatter" },
```

Comment:
why not one object instead of a list of objects?

Reply:
for future use the lodash each expression - `_.each(formatters, (formatter) => {...}`
Do you have a better solution?

Reply:
If formatters is 
```json
{
    "title1": "formatter1",
    "title2": "formatter2"
}
```

then you can do 
```js
for (let key in formatters) {
    let funcName = formatters[key];
    row[key] = this[funcName](row[key])
}
```